facade/src/main/java/com/redhat/lightblue/migrator/facade/DAOFacadeBase.java

```diff
@@ -208,6 +208,15 @@ public T call() throws Exception {
         }
     }
 
+    private Throwable extractUnderlyingException(Throwable t) {
```

Comment:
I don't want to get to the root cause, I just want to find the exception returned by the service. It may be a chain of causes, just stripped from exception wrappers added by the facade (ExecutionException and InvocationTargetException).